webapp/src/Constants.ts

```diff
@@ -50,4 +50,12 @@ export const Constants = {
         MANIFEST_PATH: '/.well-known/ai-plugin.json',
     },
     KEYSTROKE_DEBOUNCE_TIME_MS: 250,
+    MemoriesStoreTypes: {
+        Qdrant: 'Qdrant',
+        AzureCognitiveSearch: 'AzureCognitiveSearch',
+        Pinecone: 'Pinecone',
+        Milvus: 'Milvus',
+        Volatile: 'Volatile',
+        Unknown: 'Unknown',
```

Comment:
these seem like values we could/should pull from the backend? some of these options aren't currently supported (e.g. Pinecone & Milvus) and I think may confuse users. 

could we send the keys of `MemoriesStoreOptions.MemoriesStoreType` (or do something else) to communicate to the frontend what types are supported?

Reply:
I totally agree with what you suggest. In fact, I was thinking of implementing this the way you suggested.
However, I think this is something that we should have inputs from the team. On one hand, we confuse the users by showing things that we don't support (yet?). On the other hand, we want to show people that they can have all those options (in the future?).

Maybe we can create issues and mention the issues in some TODOs as we slowly add supports?

Reply:
let's bring this up in standup tmrw?

I definitely hear your point on advising the user of their options, even if we don't currently support them, I'm just not sure disabled radio buttons is the best way to do that since imo it can bloat the UX with features that we don't have a clear timeline of implementing. 
we may be able to get away with having help text that is either always visible or shown through an icon bubble, with maybe a "learn more" link as well indicating what is on the roadmap storage-wise.. not sure, Ruokan may also have opinions here.

pulling the options dynamically from the backend would allow folks to add other storage types (e.g. Chroma) without making changes to the UX code, which they may be unfamiliar with.